### DISCUSSION TITLE
Built-In post planning option

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#18753] [General]
I think it would make perfect sense for Mastodon to offer a built-in function that would allow the scheduling of Mastodon posts.

However, I think people from other Fediverse projects should work together to ensure that the ActivityPub (AP) protocol is adapted as necessary.

**Comment by lookitmychicken** [comment]
I'd like to see this too. It's a common thing in fandom for an account to regularly post images, quotes etc on a schedule. I believe it's possible to set up a bot to do this?? But it would be nice to have it built in to Mastodon.

**Comment by supernovae** [comment]
Scheduling posts inside mastodon web would be nice.
